sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs

```diff
@@ -16,6 +16,9 @@ public static class TestEnvironment
         /// <summary>The name of the environment variable from which the Form Recognizer resource's API key will be extracted for the live tests.</summary>
         public const string ApiKeyEnvironmentVariableName = "FORM_RECOGNIZER_API_KEY";
 
+        /// <summary>The name of the environment variable from which the Form Recognizer resource's API key will be extracted for the live tests.</summary>
+        public const string ContainerUrlEnvironmentVariableName = "FORM_RECOGNIZER_CONTAINER";
```

Comment:
If this is a blob container SAS URL, let's call it that instead of Container -- container could also mean Docker container in a Cognitive context.

Reply:
ohh good point. Updated